deepchecks/checks/methodology/performance_overfit.py

```diff
@@ -34,24 +34,30 @@
 
 
 class TrainTestDifferenceOverfit(TrainTestBaseCheck):
-    """Visualize overfit by displaying the difference between model metrics on train and on test data.
+    """Visualize overfit by displaying the difference between model scores on train and on test data.
 
-    The check would display the selected metrics for the training and test data, helping the user visualize
-    the difference in performance between the two datasets. If no alternative_metrics are supplied, the check would
-    use a list of default metrics. If they are supplied, alternative_metrics must be a dictionary, with the keys
-    being metric names and the values being either a name of an sklearn scoring function
-    (https://scikit-learn.org/stable/modules/model_evaluation.html#scoring) or an sklearn scoring function.
+    The check would display the selected scores for the training and test data, helping the user visualize
+    the difference in performance between the two datasets. If no 'alternative_scorers' are supplied, the check would
+    use a list of default scorers. If they are supplied, 'alternative_scorers' must be a dictionary, with the keys
+    being score names and the values being either a name of an sklearn scoring function
+    (`Scikit-learn scorers`_) or an sklearn scorer object (`Scikit-learn - defining scoring strategy`_).
 
     Args:
-        alternative_metrics (Dict[str, Callable]): An optional dictionary of metric name or scorer functions
+        alternative_scorers (Dict[str, Callable]): An optional dictionary of scorer name or scorer functions
```

Comment:
This looks good.
Lets see how it looks in the API reference (and in the auto-complete in jupyter), and then update it to be the documentation also for other classes that can receive other scorers (e.g. in boosting_overfit)